python/cable_cell_io.cpp

```diff
@@ -0,0 +1,106 @@
+#include <pybind11/pybind11.h>
+#include <pybind11/stl.h>
+
+#include <fstream>
+#include <iomanip>
+
+#include <arbor/cable_cell.hpp>
+
+#include <arborio/cableio.hpp>
+
+#include "error.hpp"
+#include "strprintf.hpp"
+
+namespace pyarb {
+
+arborio::cable_cell_component load_component(const std::string& fname) {
```

Comment:
A final suggestion here: I would like to see the option to load from a File-like object, either as an 'overload' (aka switch-on-type) or as the only interface.

This the boilerplate to read from file.
```Python
with open('fn.acc') as fd:
  comp = arb.load_component(fd)
```
 But this pattern allows to do this
```Python
text = get_acc_from_http('https://arb-db.acc/foo')
buff = StringIO(text)
comp = arb.load_component(buff)
```

Reply:
This affects most of the intefaces of arborio. Can we open a separate issue for it?

Reply:
👍 
And we should get to it quickly, it's a commonly used pattern in Python (yaml/json/... libs).